contrib/mac/framework/delete-all-rpaths.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
I know this was designed with MacOS in mind, but we may end up wanting to use this in other places, since it's useful.  Best to have this live in `contrib/delete-all-rpaths.sh`, and use `/bin/sh` instead of `bash`.